bootstrap_exercises.py

```diff
@@ -1,13 +1,14 @@
 import json
 import models
+import workout
 
-
-session = models.Session()
+session = workout.Session()
 
 exercises = json.load(open('exercises.json', 'r'))
 
 for category in exercises['exercise_categories']:
     session.add(models.ExerciseCategory(**category))
+session.commit()
```

Comment:
Yeah, so it turns out SQLAlchemy is kind of dumb. It inserts all the records into the database in no particular order. That means that if it picks an exercise before it's category has been inserted it fails because of a missing foreign key. This fixes that